onnxruntime/test/providers/cpu/reduction/reduction_ops_test.cc

```diff
@@ -924,9 +924,284 @@ TEST(ReductionOpTest, ReduceMax_default_axes_do_not_keep_dims) {
                         55.0f, 1.0f,
                         60.0f, 2.0f});
   test.AddOutput<float>("reduced", {}, {60.0f});
-  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});  // TensorRT: full reduce without keepDimensions is not supported with explicit batch                         //TensorRT: axis must be 0
+  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});// TensorRT: full reduce without keepDimensions is not supported with explicit batch //TensorRT: axis must be 0
```

Comment:
Did you change something?

Reply:
Only reformatted the comment which was badly formed before.